src/main/java/com/palantir/gradle/revapi/RevapiAcceptAllBreaksTask.java

```diff
@@ -16,33 +16,34 @@
 
 package com.palantir.gradle.revapi;
 
-import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import com.palantir.gradle.revapi.config.AcceptedBreak;
-import com.palantir.gradle.revapi.config.GradleRevapiConfig;
 import com.palantir.gradle.revapi.config.GroupNameVersion;
 import com.palantir.gradle.revapi.config.Justification;
-import java.io.File;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.util.List;
 import java.util.Set;
-import java.util.stream.Collectors;
+import org.gradle.api.DefaultTask;
+import org.gradle.api.file.RegularFileProperty;
 import org.gradle.api.provider.Property;
 import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.InputFile;
+import org.gradle.api.tasks.Nested;
 import org.gradle.api.tasks.TaskAction;
 import org.gradle.api.tasks.options.Option;
 
-public class RevapiAcceptAllBreaksTask extends RevapiJavaTask {
-    private static final ObjectMapper OBJECT_MAPPER = GradleRevapiConfig.newJsonObjectMapper();
+public class RevapiAcceptAllBreaksTask extends DefaultTask {
     public static final String JUSTIFICATION = "justification";
 
     private final Property<ConfigManager> configManager = getProject().getObjects().property(ConfigManager.class);
     private final Property<GroupNameVersion> oldGroupNameVersion =
             getProject().getObjects().property(GroupNameVersion.class);
 
+    private final RegularFileProperty resultsFile = getProject().getObjects().fileProperty();
     private final Property<Justification> justification = getProject().getObjects().property(Justification.class);
 
+    public RevapiAcceptAllBreaksTask() {
+        this.getOutputs().upToDateWhen(_task -> false);
+    }
+
+    @Nested
```

Comment:
I'd remove this, it shouldn't be involved in caching, and this task should not participate in up-to-dateness at all either (it's very cheap to deser+potentially add accepted breaks).